Woodoku/src/Entity/WoodokuBoard.py

```diff
@@ -12,17 +14,36 @@ class WoodokuBoardRepresentation:
     The first block on the bottom row is (n - 1, 0).
     The bottom-right block's coordinate is (n - 1, n - 1).
     """
-    def __init__(self):
-        pass
+    board: ndarray
+    length: int
+
+    def __init__(self, n: int):
+        self.length = n
```

Comment:
why `n` shouldn't it always be 9?

Reply:
I agree that we could have this flexibility.

Reply:
You know what, it will make the 3x3 box size be a pain-in-the-ass if we have flexible board size. So we might just fix the number.